lib/filter.js

```diff
@@ -40,7 +40,7 @@ exports.heading = function (data) {
         }, options.anchor);
 
       //  Put the anchor after the title by default, unless says otherwise
-      var link = '<a href="' + id + '" class="' + anchorOpts.style + '">"' + anchorOpts.symbol + '"</a>';
+      var link = '<a href="' + id + '" class="' + anchorOpts.style + '">' + anchorOpts.symbol + '</a>';
```

Comment:
Could you also fix the `id` problem, it's missing `#` tag, so `'<a href="' + id` should be `'<a href="#' + id`. Here is my original PR for adding the anchor: 0e09996cddec975c110d23030870b037c632e948:

```js
          if (anchorOpts.position === 'before') {
            $title.prepend(`<a href='#${id}' class='${anchorOpts.style}'>${anchorOpts.symbol}</a> `);
          } else {
            $title.append(` <a href='#${id}' class='${anchorOpts.style}'>${anchorOpts.symbol}</a>`);
          }
```

Reply:
Sure! Let me get back to you later.